### ISSUE TITLE
Add rule for extra `@param` in KDoc

### ISSUE THREAD

**Issue by kentr0w** [cqfn/diKTat#859]
Investigate why diktat doesn't trigger

```kotlin
**
 * @param x 
 * @param y 
 */
fun foo(x: Int) { }
```

**Comment by petertrr** [comment]
We have this logic here: https://github.com/cqfn/diKTat/blob/9ac8d4b516c5b03a55993a9af5e8bc7e3a0f3b70/diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocMethods.kt#L174

**Comment by petertrr** [comment]
@kentr0w please change issue title as well; also please provide a reproducer where you have encountered this issue and with which version of diktat. With the example you have provided I can see that diktat emits correct warning
